plugins/repositories/repositories_inspector.rb

```diff
@@ -90,7 +92,42 @@ def inspect_yum_repositories
       )
     end
 
-    RepositoriesScope.new(repositories)
+    RepositoriesScope.new(repositories, repository_system: "yum")
+  end
+
+  def inspect_apt_repositories
+    content = system.read_file("/etc/apt/sources.list")
+    begin
+      content += "\n" + system.run_command(
+        "bash", "-c", "cat /etc/apt/sources.list.d/*.list", stdout: :capture
+      )
+    rescue Cheetah::ExecutionFailed
+    end
+
+    RepositoriesScope.new(
+      parse_apt_cache_poliy_output(content), repository_system: "apt"
+    )
+  end
+
+  def parse_apt_cache_poliy_output(content)
```

Comment:
The name seems to be wrong, the output is the content of the `.list` files not from `apt-cache policy`.